packages/ui/package.json

```diff
@@ -24,7 +23,7 @@
     "typecheck": "tsc --noEmit --project ./tsconfig.json"
   },
   "exports": {
-    ".": "./dist-all/web.js",
+    ".": "./dist-web/web.js",
```

Comment:
Thanks for changing this to make it clearer. I think you probably know but just in case not, this will require an update in the extension's `panda.config.ts` to change the path. 

I also specify it in the `packages/ui/README.md` so it would be good to update that also.

Reply:
Yep, I have those changes on a branch in the extension, thx.

Reply:
Made the change to the readme, thought I had done that already. I pushed up the change here.